CONTRIBUTING.md

```diff
@@ -8,6 +8,11 @@ cuGraph, and all of RAPIDS in general, is an open-source project where we encour
 If you are ready to contribute, jump right to the [Contribute Code](#code) section.
 
 
+__Style Formating Tools:__
```

Comment:
should we better move this to the "Style Guide" section?

https://github.com/rapidsai/cugraph/pull/859/files#diff-6a3371457528722a734f3c51d9238c13R151

And
### Style Guild
=>
### Style Guide

Reply:
Should have pointers to the style guides

Reply:
added link